core/src/main/java/com/novoda/noplayer/PlayerSubtitleTrack.java

```diff
@@ -0,0 +1,107 @@
+package com.novoda.noplayer;
+
+public class PlayerSubtitleTrack {
```

Comment:
Cannot do that right now because its being used inside the package `exoplayer`. We have to flatten this project properly https://github.com/novoda/no-player/pull/47#discussion_r122642440